content/300-accelerate/600-faq.mdx

```diff
@@ -64,7 +64,7 @@ Under the hood, Accelerate uses Cloudflare, which uses [Anycast](https://www.clo
 
 ## How can I invalidate a cache on Accelerate?
 
-You can invalidate your cache on a project level up to five times a day. This can be done via the Cloud Project's Accelerate configuration page.
+You can invalidate your cache on a project level up to five times a day. This can be done via the Platform Console's Accelerate configuration page.
```

Comment:
I wonder if we even need "Platform Console's" here:

```suggestion
You can invalidate your cache on a project level up to five times a day. This can be done via the Accelerate configuration page.
```